include/pagmo/problem.hpp

```diff
@@ -779,6 +809,16 @@ struct prob_inner final : prob_inner_base {
     {
         return 0u;
     }
+    template <typename U, typename std::enable_if<has_c_tolerance<U>::value, int>::type = 0>
+    static vector_double get_c_tol_impl(const U &value)
+    {
+        return value.get_c_tol();
+    }
+    template <typename U, typename std::enable_if<!has_c_tolerance<U>::value, int>::type = 0>
+    static vector_double get_c_tol_impl(const U &)
+    {
+        return vector_double{};
```

Comment:
Indeed, but then we cannot call get_nec and get_nic as for the static, so we do not have the size here ...